### PR TITLE
Update node, update webpack, run audit fix

### PR DESCRIPTION
**Pull request by collinclarke**
Node 14.15 -> 18.12 (lts)
Webpack update to fix issues with 17+ node
Ran npm audit fix

### CODE REVIEW

**Comment by bpb27** [comment]
is there a good way to test this, or just CI passes so 🟢

File: webpack.config.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const config = {
     path: path.resolve(__dirname, 'dist'),
     filename: 'index.js',
     libraryTarget: 'umd',
+    hashFunction: 'xxhash64',
```
Comment (bpb27) [inline webpack.config.js:14]: who's this fella

Comment (collinclarke) [reply webpack.config.js:14]: Just copying Joe Kent's work https://github.com/ArcadiaPower/connect-react/pull/54

Comment (collinclarke) [reply webpack.config.js:14]: Looks like it's because of this https://github.com/webpack/webpack/issues/14532

**Review by bpb27 — APPROVED** [review]

**Pull request merged**